scripts/put_vectordb_model.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export ES="http://admin:${PASSWORD}@${HOSTNAME}:${PORT}"
+
+#Function to put the vectorDB model
```

Comment:
is the same comment as in https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/pull/223/files#diff-e107c0a20cdeca60fc6e456fc7e9eff8a5bafbb2fc98511848a932f7863b72d9R5
can we add more detailed comment what exactly function does

Reply:
added